test/autorest/generated/custombaseurlgroup/internal/custombaseurlgroup/paths.go

```diff
@@ -11,19 +13,18 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 )
 
-// PathsClient ...
-type PathsClient struct{}
+type PathsOperations struct{}
 
 // GetEmptyCreateRequest creates the GetEmpty request.
-func (PathsClient) GetEmptyCreateRequest(u url.URL) (*azcore.Request, error) {
+func (PathsOperations) GetEmptyCreateRequest(u url.URL, accountName string) (*azcore.Request, error) {
```

Comment:
Note that `accountName` doesn't actually get used.  Could you please open a bug to track this.

Reply:
Yes I noticed the same thing. Will do 👍